src/components/02-components/card-pending-tasks/card-pending-tasks.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import AddAlarmIcon from '@mui/icons-material/AddAlarm';
+import Link from '../../01-elements/link/link';
+
+type CardPendingTasksProps = {
+  ctaLink: string;
+};
+
+export default function CardPendingTasks({ ctaLink }: CardPendingTasksProps) {
+  const theme = useTheme();
+
+  // Styles.
+  const contentStyle = {
+    display: 'flex',
+    flexBasis: '20%',
```

Comment:
I don't think this `flexBasis` should be here. This should be added only when setting up the component in the Remix app.